.builds/freebsd.yml

```diff
@@ -15,6 +15,12 @@ environment:
 tasks:
 - setup: |
     cd Nim
+    if [ "${GITHUB_PR_NUMBER+1}" ]; then
```

Comment:
which is more portable, the version you wrote or something like `if [ -z "${GITHUB_PR_NUMBER }" ]` ?

Reply:
I think they're both much the same really, based on testing.